packages/rum-core/src/domain/rumEventsCollection/action/listenActionEvents.spec.ts

```diff
@@ -0,0 +1,115 @@
+import { createNewEvent } from '../../../../../core/test/specHelper'
+import type { OnClickCallback } from './listenActionEvents'
+import { listenActionEvents } from './listenActionEvents'
+
+describe('listenActionEvents', () => {
+  let onClickSpy: jasmine.Spy<OnClickCallback>
+  let stopListenEvents: () => void
+
+  beforeEach(() => {
+    onClickSpy = jasmine.createSpy()
+    ;({ stop: stopListenEvents } = listenActionEvents({ onClick: onClickSpy }))
+  })
+
+  afterEach(() => {
+    stopListenEvents()
+  })
+
+  it('listen to click events', () => {
+    emulateClick()
+    expect(onClickSpy).toHaveBeenCalledOnceWith(jasmine.objectContaining({ type: 'click' }), false)
+  })
+
+  describe('selection change', () => {
+    let text: Text
+
+    beforeEach(() => {
+      text = document.createTextNode('foo bar')
+      document.body.appendChild(text)
+    })
+
+    afterEach(() => {
+      document.body.removeChild(text)
+      document.getSelection()!.removeAllRanges()
+    })
+
+    it('click without selection impact should not report a selection change', () => {
+      emulateClick()
+      expect(onClickSpy.calls.mostRecent().args[1]).toBe(false)
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏in order to make the tests more readable, we could extract a helper for that, something like:
```ts
expectSelectionToHaveChanged(false)
```